index.html

```diff
@@ -614,6 +614,96 @@ <h4>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Automation
```

Comment:
I add my comment here as I do not see how your reply in Conversation page can be linked to my initial question and keep together.

> Like I said before, this is a variation on something that the Generic Sensor spec already had to deal with (the only difference here is that you also need a way to disable the override, either by allowing a special value here or adding an endpoint that listens to a DELETE http request instead of POST).

Could you clarify the overall architecture you define.
So the current set_device_posture is not enough, we need more services? And what it currently does is not enough?
So when set_device_posture reaches chrome/test/chromedriver/window_commands.cc it should somehow read and save current posture and then modifies it with setDeviceMetricsOverride(). Then we have other service clear_device_posture which reads this saved value from somewhere ("slot in top-level-traversable") I writes it using setDeviceMetricsOverride()?

> internal slot in the top-level traversable

This went totally over my head. Is "Each [top-level traversable](https://html.spec.whatwg.org/multipage/document-sequences.html#top-level-traversable) has a virtual sensor mapping, which is an [ordered map](https://infra.spec.whatwg.org/#ordered-map) of [virtual sensor types](https://www.w3.org/TR/generic-sensor/#virtual-sensor-type) to [virtual sensor](https://www.w3.org/TR/generic-sensor/#virtual-sensor)." a slot in top-level traversable? If it is where in sensor code this "slot" is handled/accessed? It try to understand how it would affect current (draft) Device Posture WebDriver code.

Reply:
I assume this is 99% wrong, but this has "clear device posture", some mods to "calculate the device posture information" and field in internal slot (assuming it is totally wrong location in spec)

https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/a9d3d5a7bfc454eb439350719c23b4a0c1935c74